requirements_pt.txt

```diff
@@ -1,3 +1,4 @@
+git+https://github.com/intel/auto-round.git@6815f8b66be456ecbef2d0beb33dbc4efeefdc04
```

Comment:
Should we update BoM with auto-round?

Reply:
yes sure, we will add the first version to be released.